### ISSUE TITLE
Deletes Users Immediately After Second Reminder

### ISSUE THREAD

**Issue by YoOoster** [Mike-on-Tour/userreminder#1]
I've set my settings that a user can be deleted 15 days after the second reminder but it seems to do it immediatly after the second reminder. 

Settings:
![iss1](https://user-images.githubusercontent.com/8483369/99018866-64e65380-255b-11eb-8d94-60f070f895c8.JPG)

Admin Log:
![iss2](https://user-images.githubusercontent.com/8483369/99018901-70397f00-255b-11eb-8c38-9531a48f07dc.JPG)

**Comment by Mike-on-Tour** [comment]
Even if I can't find anything wrong with the code your admin log seems to prove that indeed there is something wrong.
Perhaps handling the users due to deletion first might solve the problem. This has to be done in `event/main_listener.php` for which I need to know the version you are using.

**Comment by Mike-on-Tour** [comment]
Since I have no clue what the reason might be I have changed the sequence in which users are reminded and deleted in order to avoid checking for users to be deleted right after they have been reminded the second time.
Since it is not clear what version was used I prepared two versions of the script doing the automatic reminding and deleting,
one for **Userreminder 1.2.1**:
[main_listener_1.2.1a.zip](https://github.com/Mike-on-Tour/userreminder/files/5540496/main_listener_1.2.1a.zip)
and one for **Userreminder 1.3.x**:
[main_listener_1.3.1a.zip](https://github.com/Mike-on-Tour/userreminder/files/5540497/main_listener_1.3.1a.zip)

Please unzip the correct version and copy it via FTP to the `ext/mot/userreminder/event` directory overwriting the existing file.

This issue is also available at [mike-on-tour.com](https://www.mike-on-tour.com/viewtopic.php?f=5&t=12)

If this did the trick please inform me in order to be able to close this issue.

**Comment by YoOoster** [comment]
I recently copied over my forum from an vps to a dedicated server. After posting here and upgrading my extensions to the latest versions it seemed to break another extension that was using time/timezone. I had to manually set my server and php.ini file to the correct timezone to fix that extension and afterwards yours was fixed as well.

So either:
It was fixed by updating to 1.3.1. 
It was fixed by changing server & php time/timezone settings.

I didn't used your edited files.
